phalcon/cache/backend/redis.zep

```diff
@@ -188,18 +188,19 @@ class Redis extends Backend
 	 * @param int|string keyName
 	 * @param string content
 	 * @param int lifetime
+	 * @param boolean stopBuffer
 	 */
 	public function save(keyName = null, content = null, lifetime = null, boolean stopBuffer = true) -> boolean
 	{
-		var nonePrefixedKey, lastKey, frontend, redis, cachedContent, preparedContent,
+		var prefixedKey, frontend, redis, lastKey, cachedContent, preparedContent,
 			tmp, ttl, success, options, specialKey, isBuffering;
 
 		if keyName === null {
-			let lastKey = this->_lastKey;
-			let nonePrefixedKey = substr(lastKey, 5 + strlen(this->_prefix));
+			let prefixedKey = substr(lastKey, 5),
```

Comment:
Now the number 5 is more than meaningless. The stats key may be `_PHCR` (5) or just `_` (1) right?

Reply:
On other hand there is still at least 2 places where we require exact `_PHCR`

Reply:
Made changes to take into account the length of the `statsKey`